css_tasks/task1/index.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Task1</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body topmargin="0" leftmargin="0">
+    <table width="100%" border="1px" cellpadding="0" cellspacing="0">
+      <!-- Header -->
+      <thead>
+        <th colspan="3" align="center" height="96px">
+          <font size="5" face="Montserrat" color="red"
+            >Markup of "Basic HTML" Task</font
+          >
+        </th>
+      </thead>
+      <tbody>
+        <tr>
+          <!-- Left Sidebar -->
+          <td width="198px" valign="top">
+            <table cellpadding="5px" width="100%">
+              <tbody>
+                <th height="40px" align="left">
+                  <font size="5" face="Montserrat">Menu</font>
+                </th>
+                <tr>
+                  <td>
+                    <font face="Montserrat"><a href="#">Main</a></font>
```

Comment:
Is it not enough setting font only for root element?

Reply:
Hm.., probably it's stupid to admit, but I can't figure out how I can use font tag on root. Never used font tag before :)